demo/kserve/scripts/README.md

```diff
@@ -1,55 +1,63 @@
-# Script-based Installation
+# Script-based installation of Kserve and dependencies
 
-## Clone the repo
+Note: You have the alternative option of installing the KServe/Caikit/TGIS stack by using [Step-by-step commands](demo/kserve/install-manual.md).
 
+**Prerequisites**
 
-~~~
-git clone https://github.com/opendatahub-io/caikit-tgis-serving
-cd caikit-tgis-serving/demo/kserve
-~~~
+- To support inferencing, your cluster needs a node with 4 CPUs and 8 GB memory.
+- You have cluster administrator permissions.
+- You have installed the OpenShift CLI (`oc`).
 
-**NOTE:** If you would like to use brew, please create a credential secret and ICSP.
+**Procedure**
 
-## Set the environment variables
+1. Clone the `caikit-tgis-serving` repo.
+   ~~~
+   git clone https://github.com/opendatahub-io/caikit-tgis-serving
+   cd caikit-tgis-serving/demo/kserve
+   ~~~
 
-If the variables below are not set, the script will ask for them.
+2. Set environment variables.
 
+   Note: If you do not set values for these variables before you run the script, the script asks for them.
 
-- Brew is a registry where WIP images are published. Please look for the desired/correct brew tag, as it changes with every build. 
-~~~
-export TARGET_OPERATOR=brew
-export BREW_TAG=554169
-~~~
+   **TARGET_OPERATOR:** Possible values are `odh` or `rhods`. For example:
+   - Red Hat OpenSHift Data Science (1.32+)
+     ~~~
+     export TARGET_OPERATOR=rhods
+     ~~~
 
-- RHODS (1.32+)
-~~~
-export TARGET_OPERATOR=rhods
-~~~
+   - Open Data Hub (1.9+)
```

Comment:
it should be Open Data Hub (1.9+) but on operator v2.x version.
operator v1.x does not support Kserve